doc/tiered_support.md

```diff
@@ -1,196 +1,163 @@
+| x86_64-linux-cosmo        | N/A                                                                                                   | Target             |
+| x86_64-unknown-freebsd    | N/A                                                                                                   | Target             |
+| x86_64-wrs-vxworks        | N/A                                                                                                   | Target             |
+| xtensa-esp32-elf          | N/A                                                                                                   | Target             |
+| AliOS compatibility       | N/A                                                                                                   | Portability        |
+| Android Compatibility     | N/A                                                                                                   | Portability        |
+| Cosmo Compatibility       | N/A                                                                                                   | Portability        |
+| FreeBSD Compatibility     | N/A                                                                                                   | Portability        |
+| iOS Compatibility         | N/A                                                                                                   | Portability        |
+| RIOT OS Compatibility     | N/A                                                                                                   | Portability        |
+| RT-Thread Compatibility   | N/A                                                                                                   | Portability        |
+| VxWorks Compatibility     | N/A                                                                                                   | Portability        |
+| Windows Compatibility     | N/A                                                                                                   | Portability        |
+| Legacy Exception Handling | [WAMR_BUILD_EXCE_HANDLING](./build_wamr.md#exception-handling)                                        | Wasm Proposal      |
+| Multi-memory              | [WAMR_BUILD_MULTI_MEMORY](./build_wamr.md#multi-memory)                                               | Wasm Proposal      |
+| Multi-tier JIT            | [Combination of flags](./build_wamr.md#configure-multi-tier-jit)                                      | Runtime Extensions |
+| AoT Validator             | [WAMR_BUILD_AOT_VALIDATOR](./build_wamr.md#aot-validator)                                             | Runtime Extensions |
+| Bulk-memory-opt           | [WAMR_BUILD_BULK_MEMORY_OPT](./build_wamr.md#bulk-memory-opt)                                         | Runtime Extensions |
+| Call-indirect-overlong    | [WAMR_BUILD_CALL_INDIRECT_OVERLONG](./build_wamr.md#call-indirect-overlong)                           | Runtime Extensions |
```

Comment:
WAMR_BUILD_BULK_MEMORY_OPT and WAMR_BUILD_CALL_INDIRECT_OVERLONG seem to have duplicated entries in B and C.
i guess they should be in the same tier as WAMR_BUILD_LIME1.